packages/menu-bar/src/vaadin-menu-bar-submenu.js

```diff
@@ -53,6 +60,19 @@ class MenuBarSubmenu extends ContextMenu {
     return 'vaadin-menu-bar';
   }
 
+  /**
+   * @param {DocumentFragment} dom
+   * @return {null}
```

Comment:
**nit:** The method doesn't seem to return `null`.

Reply:
Thanks, changed to `ShadowRoot` which is the correct type.